schedule/functional/extra_tests_in_textmode.yaml

```diff
@@ -0,0 +1,75 @@
+name:           extra_tests_in_textmode
+description:    >
+    Maintainer: slindomansilla.
+    Extra tests about software in Basesystem and Server Application modules
+schedule:
+    - boot/boot_to_desktop
+    - console/prepare_test_data
+    - console/consoletest_setup
+    - console/zypper_lr_validate
+    - console/zypper_ref
+    - console/zypper_info
+    - console/validate_packages_and_patterns
+    - console/zypper_extend
+    - console/check_os_release
+    - console/openvswitch
+    - console/pam
+    - console/sshd
+    - console/update_alternatives
+    - console/rpm
+    - console/slp
+    - console/aplay
+    - console/wavpack
+    - console/libvorbis
+    - console/command_not_found
+    - console/openssl_alpn
+    - console/autoyast_removed
+    - console/cron
+    - console/syslog
+    - console/mta
+    - console/yast2_lan_device_settings
+    - console/check_default_network_manager
+    - console/git
+    - console/cups
+    - console/java
+    - console/sqlite3
+    - console/gdb # should go to extra_tests_in_textmode_sdk
+    - console/sysctl
+    - console/sysstat
+    - console/curl_ipv6
+    - console/wget_ipv6
+    - console/ca_certificates_mozilla
+    - console/unzip
+    - console/machinery # should go to extra_tests_in_textmode_phub
+    - console/gpg
+    - console/rsync
+    - console/clamav
+    - console/shells
+    - console/sudo
+    - console/dstat
+    - x11/evolution/evolution_prepare_servers
```

Comment:
Is it really needed here? or it's just a test that's there to support mutt? (I.e, setting up a mailserver?)

Reply:
No idea, I just copy-pasted the list of current scheduled modules by main.pm
I am open to unschedule any module.